spec/mailers/enterprise_mailer_spec.rb

```diff
@@ -2,15 +2,27 @@
 
 describe EnterpriseMailer do
   let!(:enterprise) { create(:enterprise) }
+  let!(:user) { create(:user) }
 
   before do
     ActionMailer::Base.deliveries = []
   end
 
-  it "should send a welcome email when given an enterprise" do
-    EnterpriseMailer.welcome(enterprise).deliver
-    ActionMailer::Base.deliveries.count.should == 1
-    mail = ActionMailer::Base.deliveries.first
-    expect(mail.subject).to eq "#{enterprise.name} is now on #{Spree::Config[:site_name]}"
+  describe "#welcome" do
+    it "should send a welcome email when given an enterprise" do
+      EnterpriseMailer.welcome(enterprise).deliver
+      ActionMailer::Base.deliveries.count.should == 1
```

Comment:
:warning:

Reply:
In my defence, I didn't write that `should`, I indented that section and didn't chcek it all. :grimacing: